handler.py

```diff
@@ -17,8 +17,11 @@
 DEFAULT_TTL = 60 * 60 * 24
 
 # Environment Constants
-S3_BUCKET = os.environ.get("S3_BUCKET")
-DDB_TABLE = os.environ.get("DDB_TABLE")
+S3_BUCKET = os.environ.get("S3_BUCKET", "pushbox_test")
+DDB_TABLE = os.environ.get("DDB_TABLE", "pushbox_test")
+# use stage as default.
+# prod = oauth.accounts.firefox.com
+FXA_HOST = os.environ.get("FXA_VERIFY_HOST", "oauth.stage.mozaws.net")
```

Comment:
It may be worth defaulting to "stable dev" here, because the FxA dev team are able to help debug on that box, rather than having to loop in ops if something goes pear-shaped: https://oauth-stable.dev.lcip.org/